buildkite/bazel-bench/bazel_bench.py

```diff
@@ -86,23 +111,29 @@ def _get_bazel_commits(date, bazel_repo_path):
     ]
     command_output = subprocess.check_output(args, cwd=bazel_repo_path)
     decoded = command_output.decode("utf-8").split("\n")
```

Comment:
If applicable, `.splitlines()` is usually a better choice than `.split("\n")` because it gives a more consistent result (see the lower two examples in the docs here: https://docs.python.org/3/library/stdtypes.html#str.splitlines).

Reply:
Done. TIL.